src/core/libraries/kernel/thread_management.cpp

```diff
@@ -1086,6 +1086,16 @@ int PS4_SYSV_ABI scePthreadAttrGetstack(ScePthreadAttr* attr, void** addr, size_
     return SCE_KERNEL_ERROR_EINVAL;
 }
 
+int PS4_SYSV_ABI scePthreadAttrSetstack(ScePthreadAttr* attr, void* addr, size_t size) {
+    int result = pthread_attr_setstack(&(*attr)->pth_attr, addr, size);
```

Comment:
Before calling this function, we need to ensure that:
* `attr` is a valid pointer
* `stack_addr` is a valid pointer
* `size` is less than specified by PS4 system maximum